clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/SharedKafkaConsumer.java

```diff
@@ -219,11 +239,12 @@ protected void waitAfterUnsubscribe(long currentPollTimes, Set<PubSubTopicPartit
       LOGGER.info("Wait for poll request in `unsubscribe` function got interrupted.");
       Thread.currentThread().interrupt();
     }
+    stats.recordTotalWaitAfterUnsubscribeLatency((int) (time.getMilliseconds() - startTimeMs));
```

Comment:
This will overflow right? time.getMilliseconds if it's returning the system epoch time in millis will overflow the integer.  I think we should just use long's here

Reply:
Oh, the -, I guess that wouldn't overflow.